src/ai/backend/manager/api/vfolder.py

```diff
@@ -3296,14 +3446,13 @@ def create_app(default_cors_options):
     vfolder_resource = cors.add(app.router.add_resource(r"/{name}"))
     cors.add(vfolder_resource.add_route("GET", get_info))
     cors.add(vfolder_resource.add_route("DELETE", delete_by_name))
+    cors.add(add_route("GET", r"/{name}/id", get_vfolder_id))
```

Comment:
I think it would be better to move the `name` parameter to a query param.

Since `/folders/{name}` may conflict with `/folders/{invitations,purge,restore-from-trash-bin,delete-from-trash-bin}`, I'd suggest using a new app prefix: `/vfolders/` for new UUID-based API routes.

Reply:
Let's postpone this refactoring to a follow-up PR.